addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -370,6 +370,57 @@ options.Class.include({
             },
         });
     },
+    /**
+     * @private
+     * @param {String} type either 'head' or 'footer'
+     */
+    async openCustomCodeDialog(previewMode, widgetValue, params) {
```

Comment:
Mmh, not used to used that new ES6 syntax but I guess this won't do any harm now. It was breaking in tests before 13.0.